### PR TITLE
Show nullable annotations in tuple signature help

### PR DESCRIPTION
**Pull request by jasonmalinowski**
Fixes https://github.com/dotnet/roslyn/issues/37252

### CODE REVIEW

File: src/Workspaces/Core/Portable/Utilities/NullableHelpers/TypeSymbolWithNullableAnnotation.cs [hunk 1]
```diff
@@ -187,7 +187,9 @@ public ImmutableArray<SymbolDisplayPart> ToMinimalDisplayParts(SemanticModel sem
 
             public ImmutableArray<SymbolDisplayPart> ToMinimalDisplayParts(SemanticModel semanticModel, int position, SymbolDisplayFormat format = null)
             {
-                return WrappedSymbol.ToMinimalDisplayParts(semanticModel, position, format);
+                // Call the right API once https://github.com/dotnet/roslyn/pull/35698 is merged
```
Comment (CyrusNajmabadi) [inline src/Workspaces/Core/Portable/Utilities/NullableHelpers/TypeSymbolWithNullableAnnotation.cs]: i trust you here :)

Comment (jasonmalinowski) [reply src/Workspaces/Core/Portable/Utilities/NullableHelpers/TypeSymbolWithNullableAnnotation.cs]: I'm not sure I trust myself. This is my fault for not getting the compiler API in earlier, but at this point that's locked down for this release and for _that_ PR doesn't meet the bar. This is still worth fixing though.

Comment (ryzngard) [reply src/Workspaces/Core/Portable/Utilities/NullableHelpers/TypeSymbolWithNullableAnnotation.cs]: Why are we using flow state at all? Why can't this just take a NullableAnnotation?

Comment (jasonmalinowski) [reply src/Workspaces/Core/Portable/Utilities/NullableHelpers/TypeSymbolWithNullableAnnotation.cs]: We forgot to add the overload, and now the compiler is locked and adding it (which I did in that other PR but then didn't merge) will take too much work. So I'll replace this with the real implementation in 16.4.

**Pull request merged**